packages/dashboard/src/components/WalletQRCodeContainer/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { memo } from 'react'
+import { experimentalStyled as styled } from '@material-ui/core'
+
+const QRCodeContainer = styled('div')(
+    ({ width, height, borderWidth, borderHeight }: WalletQRCodeProps) => `
+    width: ${width}px;
+    height: ${height}px;
+    background: linear-gradient(to right, black ${borderHeight}px, transparent ${borderHeight}px) 0 0,
```

Comment:
Emm I think for this effect, this CSS is quite scary. Maybe an SVG can be better.

Reply:
hmm maybe this is fine. never mind